frontend/src/components/HostMetricsDetail/index.tsx

```diff
@@ -0,0 +1,154 @@
+import './HostMetricsDetail.styles.scss';
+
+import { Color, Spacing } from '@signozhq/design-tokens';
+import { Drawer, Progress, Radio, Tag, Typography } from 'antd';
+import { RadioChangeEvent } from 'antd/lib';
+import { useIsDarkMode } from 'hooks/useDarkMode';
+import {
+	BarChart2,
+	ChevronsLeftRight,
+	DraftingCompass,
+	Package2,
+	ScrollText,
+	X,
+} from 'lucide-react';
+import { useState } from 'react';
+
+import { VIEW_TYPES, VIEWS } from './constants';
+import { HostDetailProps } from './HostMetricDetail.interfaces';
+
+// import Metrics from './Metrics';
+// import Overview from './Overview';
+
```

Comment:
can you remove the commented code?

Reply:
@ahmadshaheer this has been fixed in other PR.